mainnet/validators/48Club.toml

```diff
@@ -0,0 +1,6 @@
+Address = "0x0"
```

Comment:
the address is empty?

Reply:
Yes, because we use this sole interface for all 48Club partner validators, so we can not provide a fixed address.

Instead we have a mev_listCoinbase endpoint to allow dynamic query.